README.md

```diff
@@ -1,3 +1,5 @@
+# Zendesk Support Modeling dbt Package ([Docs](https://fivetran.github.io/dbt_zendesk/))
```

Comment:
Should we move the H1 heading in the `dbt_zendesk_source` package too? No worries if it's out of scope.

Reply:
yes good reminder